pkg/cmd/server/api/types.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
+const (
+	FeatureBuilder    = `Builder`
```

Comment:
We shouldn't use spaces in our constants - these should be constants like all our other API constants - "Builder", "S2IBuilder", "WebConsole"